client/src/components/WriteMessage.jsx

```diff
@@ -55,42 +53,22 @@ class WriteMessage extends React.Component {
     })
   }
 
-  // retrieveMessages() {
-  //   fetch(`/messages/${this.props.userId}/?receiver_id=${this.props.receiverId}`, {
-  //     method: 'GET',
-  //     headers: new Headers({
-  //       'Content-Type': 'application/json',
-  //       'Authorization': `Bearer ${sessionStorage.getItem('authToken')}`,
-  //     })
-  //   })
-  //   .then((response) => {
-  //     if (!response.ok) {
-  //       throw Error('failed to retrieve messages...')
-  //     }
-  //     return response.json();
-  //   })
-  //   .then((data) => {
-  //     console.log('messages retrieved! data: ', data);
-  //   })
-  //   .catch((error) => {
-  //     console.log('retrieveMessages failed! error: ', error);
-  //   })
-  // }
-
-  // componentDidMount() {
-  //   setTimeout(this.retrieveMessages, 1500);
-  // }  
+  scrollToBottom() {
+    const node = ReactDOM.findDOMNode(this.messagesEnd);
```

Comment:
i believe it does, I put a div that will always be at the bottom of the chat log so when the component updates, itll jump straight to it in the chat window